ci-operator/step-registry/cucushift/installer/check/ibmcloud/custom-endpoints/cucushift-installer-check-ibmcloud-custom-endpoints-ref.yaml

```diff
@@ -42,3 +42,23 @@ ref:
     default: ""
     documentation: |-
       GlobalTagging endpoint
+  - name: SERVICE_ENDPOINT_COSConfig
+    default: ""
+    documentation: |-
+      COSConfig endpoint
+  - name: SERVICE_ENDPOINT_GlobalCatalog
+    default: ""
+    documentation: |-
+      GlobalCatalog endpoint
+  - name: SERVICE_ENDPOINT_KeyProtect
+    default: ""
+    documentation: |-
+      KeyProtect endpoint
+  - name: SERVICE_ENDPOINT_CIS
```

Comment:
I did not see the default endpoint for CIS defined in eps_default.json, if we are not ready to test it, pls remove it.